tests/integration-tests/utils.py

```diff
@@ -194,6 +227,27 @@ def retrieve_cfn_resources(stack_name, region):
     return resources
 
 
+def get_cfn_events(stack_name, region):
+    """Retrieve CloudFormation Stack Events from a give stack."""
+    if not stack_name:
+        logging.warning("stack_name not provided when retrieving events")
+        return []
```

Comment:
I am conflicted on this - would it be better to use `yield from []` and then return here for consistency?